### PR TITLE
fix(cdk)!: the `tuiLet` directive can possibility emit of null

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

### CODE REVIEW

**Review by nsbarsukov — APPROVED** [review]

File: projects/demo/src/modules/tables/table/examples/4/index.html [hunk 1]
```diff
@@ -74,10 +74,10 @@
         </thead>
         <tbody
             tuiTbody
-            [data]="data"
+            [data]="data ?? []"
```
Comment (waterplea) [inline projects/demo/src/modules/tables/table/examples/4/index.html:77]: Probably can be `ngIf` instead

File: projects/kit/components/input-date-range/input-date-range.template.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
         [pseudoActive]="pseudoActive"
         [invalid]="computedInvalid"
         [nativeId]="nativeId"
-        [filler]="getComputedRangeFiller(dateFiller)"
+        [filler]="getComputedRangeFiller(dateFiller ?? '')"
```
Comment (waterplea) [inline projects/kit/components/input-date-range/input-date-range.template.html:20]: `||`

File: projects/addon-editor/components/toolbar-tools/list-configs/list-configs.template.html [hunk 3]
```diff
@@ -63,7 +63,7 @@
                     appearance="icon"
                     class="t-option"
                     [tuiHint]="texts.outdent"
-                    [disabled]="!listState.ordered && !listState.unordered"
+                    [disabled]="(!listState?.ordered && !listState?.unordered) || false"
```
Comment (waterplea) [inline projects/addon-editor/components/toolbar-tools/list-configs/list-configs.template.html:66]: `!!`

File: projects/addon-editor/components/toolbar-tools/list-configs/list-configs.template.html [hunk 4]
```diff
@@ -52,7 +52,7 @@
                     appearance="icon"
                     class="t-option t-option_margin"
                     [tuiHint]="texts.indent"
-                    [disabled]="!listState.ordered && !listState.unordered"
+                    [disabled]="(!listState?.ordered && !listState?.unordered) || false"
```
Comment (waterplea) [inline projects/addon-editor/components/toolbar-tools/list-configs/list-configs.template.html:55]: `!!`

File: projects/addon-editor/components/toolbar-tools/list-configs/list-configs.template.html [hunk 5]
```diff
@@ -41,7 +41,7 @@
                     appearance="icon"
                     class="t-option t-option_margin"
                     [tuiHint]="texts.orderedList"
-                    [pseudoActive]="listState.ordered"
+                    [pseudoActive]="listState?.ordered ?? false"
```
Comment (waterplea) [inline projects/addon-editor/components/toolbar-tools/list-configs/list-configs.template.html:44]: `!!`

File: projects/addon-editor/components/toolbar-tools/list-configs/list-configs.template.html [hunk 6]
```diff
@@ -30,7 +30,7 @@
                     appearance="icon"
                     class="t-option t-option_margin"
                     [tuiHint]="texts.unorderedList"
-                    [pseudoActive]="listState.unordered"
+                    [pseudoActive]="listState?.unordered ?? false"
```
Comment (waterplea) [inline projects/addon-editor/components/toolbar-tools/list-configs/list-configs.template.html:33]: `!!`

File: projects/addon-editor/components/toolbar-tools/align-content/align-content.template.html [hunk 7]
```diff
@@ -50,7 +50,7 @@
                     icon="tuiIconAlignRightLarge"
                     appearance="icon"
                     class="t-option t-option_margin"
-                    [pseudoActive]="alignState.right"
+                    [pseudoActive]="alignState?.right ?? false"
```
Comment (waterplea) [inline projects/addon-editor/components/toolbar-tools/align-content/align-content.template.html:53]: `!!`

File: projects/addon-editor/components/toolbar-tools/align-content/align-content.template.html [hunk 8]
```diff
@@ -39,7 +39,7 @@
                     icon="tuiIconAlignCenterLarge"
                     appearance="icon"
                     class="t-option t-option_margin"
-                    [pseudoActive]="alignState.center"
+                    [pseudoActive]="alignState?.center ?? false"
```
Comment (waterplea) [inline projects/addon-editor/components/toolbar-tools/align-content/align-content.template.html:42]: `!!`

File: projects/addon-editor/components/toolbar-tools/align-content/align-content.template.html [hunk 9]
```diff
@@ -28,7 +28,7 @@
                     icon="tuiIconAlignLeftLarge"
                     appearance="icon"
                     class="t-option t-option_margin"
-                    [pseudoActive]="alignState.left"
+                    [pseudoActive]="alignState?.left ?? false"
```
Comment (waterplea) [inline projects/addon-editor/components/toolbar-tools/align-content/align-content.template.html:31]: `!!`

**Review by waterplea — CHANGES_REQUESTED** [review]

**Review by waterplea — APPROVED** [review]

**Pull request merged**